content/docs/getting-started/hello-world.md

```diff
@@ -1,150 +1,154 @@
 ---
-title: "Hello World!"
+title: "Hello world!"
 weight : 1010
 menu:
   docs:
     parent: getting_started
 ---
 
-To kick off our tour of Tokio, we will start with the obligatory "hello world"
-example. This program will create a TCP stream and write "hello, world!" to the stream.
-The difference between this and a Rust program that writes to a TCP stream without Tokio
-is that this program won't block program execution when the stream is created or when
-our "hello, world!" message is written to the stream.
+This guide assumes that a knowledge of the [Rust programming language] and are
+using Rust `1.39.0` or higher. To use Tokio with earlier versions of Rust,
+please check out the [Tokio 0.1 docs].
 
-Before we begin you should have a very basic understanding of how TCP streams
-work. Having an understanding of Rust’s [standard library
-implementation](https://doc.rust-lang.org/std/net/struct.TcpStream.html) is also
-helpful.
+To check version of Rust on the command line:
+```bash
+rustc --version
+rustc 1.39.0 (4560ea788 2019-11-04)
+```
 
-Let's get started.
+In version `1.39.0`, [Rust introduced async-await], see the [Async Book] for
+in-depth documentation of these new Rust language features. This guide also
+seeks to provide a practical, hands-on introduction for all of the Rust language
+features that are needed for programming with Tokio.
 
-First, generate a new crate.
+Also, most readers will have some experience with writing networking code
+using the [Rust standard library] or another language, though folks who are
+new to network programming should be able to follow along.
 
-```bash
-$ cargo new hello-world
-$ cd hello-world
-```
+# Introducing asynchronous programming
 
-Next, add the necessary dependencies in `Cargo.toml`:
+As our first introduction to [asynchronous programming](../overview) with
+Tokio, we will start with a tiny app that sends "hello, world!" over a
+network connection.
 
-```toml
-[dependencies]
-tokio = { version = "0.2", features = ["full"] }
-```
+The difference between Tokio and a Rust program written using the standard
+library is that Tokio has the flexibility to block program execution
```

Comment:
This doesn't read right to me. It sounds like we're saying that Tokio's advantage is that it blocks your program. Which a) doesn't sound like a good thing; and b) not sure that's the case. It only blocks execution within the async function, not the whole program, right?

Reply:
oops -- missed "not" changing to "has the flexibility to avoid blocking program execution"